docs/external-node.md

```diff
@@ -561,3 +562,26 @@ interfaces will be added in the future.
 
 `ExternalNode` name must be unique in the `cluster` scope even though it is
 itself a Namespaced resource.
+
+## Known issues
+
+OVS configurations must be manually removed after the VM is rebooted, and then
+a manual restart on antrea-agent is needed. This is a workaround to resolve
```

Comment:
mention what would happen if VM is reboot and what wont work

Reply:
Added,